data_processing/eia.py

```diff
@@ -0,0 +1,1526 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+path = '../../data/eia_generation_'
```

Comment:
```suggestion
path = "../../data/"
fileprefix = "eia_generation_"
```
We don't want the path to include a file name because it's not really a path. I.e. I can't point to a different folder with the datafiles.